lib/msal-core/test/UserAgentApplication.spec.ts

```
@@ -53,6 +53,15 @@ describe("UserAgentApplication", function () {
 
     let msal: UserAgentApplication;
 
+    let authCallback = function (error, response) {
+        if (error) {
+            console.log("Error: " + error);
```

why have console logs in spec files?

---

More of a placeholder than anything, these only get called if there is any issue with the tests or the tests fail.

---

can we just throw the error so it shows up?